src/docs/product/alerts-notifications/index.mdx

```diff
@@ -6,26 +6,40 @@ redirect_from:
 description: "Sentry categorizes notifications into alerts and other types of notifications. Learn more about how these are managed."
 ---
 
-Sentry categorizes notifications broadly into alerts and non-alert notifications.
+Alerts notify you about errors and performance problems in your application. Notifications send emails for situations like an issue state change or if a release has deployed. Alerts and notifications are customizable to your organization's needs with options to pair with [integrations](/product/integrations/), flexible configurations, and choices for both error or performance alerting. Default settings for notifications are also available.
 
-## Alerts
+## Issue Alerts for Errors
 
-[Alerts](/product/alerts-notifications/alerts/) notify you about problems in your application and can be sent to many supported integrations. You can set alerts on both errors and performance data.
+Issue alerting sets alerts on changes within grouped error events, or _issues_. Issues are grouped automatically by Sentry based on stack trace and other factors. Create alerts for issue-level changes, such as:
 
-- **Errors**: Sentry automatically groups individual error events into Issues based on stack trace and other factors. You can then set alerts on changes in those issues (such as new issues or issues affecting many users), or on the raw events (such as frequency of events with a given tag).
+- New issues
+- Issue frequency increasing
+- Resolved and ignored issues getting unresolved
 
-- **Performance**: You can alert on performance metrics like latency, apdex, failure rate, and throughput.
+To change the frequency of your alerts, configure your [Alert Settings](/product/alerts-notifications/alert-settings/).
 
-For more details, see the [full documentation on Alerts](/product/alerts-notifications/alerts/).
+## Metric Alerts for Errors & Performance
 
-## Non-Alert Notifications
+[Metric alerts](/product/alerts-notifications/alerts/#metric-alerts) monitor macro-level [metrics](/product/performance/metrics/) for both error events and transaction events. A metric takes a set of events and computes an aggregate value using a function such as `count()` or `avg()` applied to the event properties over a period of time. Metric alerting is triggered when a [threshold](/product/performance/metrics/#p50-threshold) is breached. You can set the threshold for metrics using the value of an aggregate function to see how errors are impacting your user's experience, the [performance](/product/performance/) speed of your application, and more.
```

Comment:
"You can set the threshold for metrics using the value of an aggregate function to see how errors are impacting your user's experience, the performance speed of your application, and more." I don't understand this sentence, would suggest removing it as it doesn't seem to add value